runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollPlugin.java

```diff
@@ -201,6 +219,12 @@ public void onGameTick(final GameTick event)
 			{
 				GameObjectQuery query = new GameObjectQuery().idEquals(objectId);
 				objectsToMark = queryRunner.runQuery(query);
+
+				// Set hint arrow to first object found as there can only be 1 hint arrow
+				if (objectsToMark.length >= 1)
+				{
+					client.setHintArrow(objectsToMark[0].getWorldLocation());
```

Comment:
A check can be added to loop through objectsToMark to find an object location that matches the clue.getLocation()

Reply:
This comment should be probably on #1631

Reply:
I'll move it there. Didn't realize this was from a dependency.